test/safe_math/safe_math.js

```diff
@@ -0,0 +1,122 @@
+const BigNumber = require('bn.js');
+const SafeMathTest = artifacts.require('SafeMathTest');
+const utils = require('../test_lib/utils');
+
+let safeMathInstance;
+contract('SafeMath', function () {
+	const MAX_UINT = new BigNumber(2).pow(new BigNumber(256)).sub(new BigNumber(1));
+	
+	beforeEach(async function () {
+		
+		safeMathInstance = await SafeMathTest.new();
+		
+	});
+	
+	describe('add', function () {
+		it('adds correctly', async function () {
+			const a = new BigNumber(5678);
+			const b = new BigNumber(1234);
+			assert((await safeMathInstance.add.call(a,b)).eq(a.add(b)));
```

Comment:
Why "safeMathInstance" and not "safeMath"? A quick GH search through openst-contracts (and mosaic-contracts) suggests this is a departure from typical practice. Is there a reason for this departure (e.g., "Instance" signifies something special here that is not the case for the other tests)?